keyboards/opticalkb/rev1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,92 @@
+// Copyright 2022 Girish Palya (@girishji)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+bool     is_cmd_grv_active = false;
```

Comment:
Default keymap should be kept simple, as this is what is copied when using the new keymap command, and overly complex keymaps can be confusing for newer users. 

Creating a "stock" type keymap that has the more advanced features is fine. though.

Reply:
Thanks for suggestions. Requested changes are committed.